jetty-unixdomain-server/src/main/config/modules/unixdomain.mod

```diff
@@ -0,0 +1,36 @@
+[description]
+Enables support for Java 16 Unix-Domain server sockets.
```

Comment:
```suggestion
Enables support for HTTP over Java 16 Unix-Domain server sockets.
```
rename module to `unixdomain-http` or `http-unixdomain`

Reply:
Done.